docs/sources/reference/components/remote.aws.secrets_manager.md

```diff
@@ -0,0 +1,92 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/remote.aws.secrets_manager/
+description: Learn about remote.aws.secrets_manager
+title: remote.aws.secrets_manager
+---
+
+# remote.aws.secret_manager
+
+`remote.aws.secrets_manager` securely exposes value of secrets located in [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/) to other components.
+By default, the secret would be fetched one time only at startup. If configured, the secret will be polled for changes so that the most recent value is always available. 
```

Comment:
```suggestion
By default, the secret is fetched once only at startup. If configured, the secret is polled for changes so that the most recent value is always available. 
```